Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -161,3 +209,89 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
         }
     }
 }
+
+// MARK: Dictionary Colon Rule
+extension ColonRule {
+
+    public func validateFile(_ file: File, kind: SwiftExpressionKind,
```

Comment:
that was the reason I put that in an extension, to group the dictionary-related functions. I'll add a comment either way 👍